tailwind.config.js

```diff
@@ -34,7 +34,8 @@ export default {
 			platinum: '#e4e4e4',
 			'mountain-meadow': '#30af91',
 			'dodger-blue': '#1e90ff',
-			turquoise: '#59e7d6'
+			turquoise: '#59e7d6',
+			'alice-blue': '#ecf3fb'
```

Comment:
Unrelated: At some point we should have a look if all colors are still used. It feels like there will be some that can be cleaned.

Reply:
Yes, I was having a look if there is some lint-ish rule that takes care of it for CSS, but the closest one is not that good. I will have a better look after the remodelling.

In the meanwhile, I can go just searching usage eheh